### ISSUE TITLE
How to build this module for dynamically loading to nginx?

### ISSUE THREAD

**Issue by xcellenceit** [FRiCKLE/ngx_cache_purge#72]
Does loading this module dynamically is supported?

Moreover, how do we build it for using dynamically?

**Comment by dvershinin** [comment]
If you're on CentOS/RHEL 6, 7, or 8, the easiest way to install the dynamic module (without building) is through the GetPageSpeed repository (commercial) using [instructions](https://nginx-extras.getpagespeed.com/modules/cache-purge/):

```bash
yum -y install https://extras.getpagespeed.com/release-latest.rpm
yum -y install nginx-module-cache-purge
```

Enable the module by adding the following at the top of /etc/nginx/nginx.conf:

```nginx
load_module modules/ngx_http_cache_purge_module.so;
```

**Comment by Danrancan** [comment]
@xcellenceit
> Does loading this module dynamically is supported?

Yes, you can find the dynamic version in my fork at https://github.com/Danrancan/ngx_cache_purge_dynamic.

> Moreover, how do we build it for using dynamically?

The dynamic build instructions are on my forks README.
